packages/dds/legacy-dds/src/array/sharedArray.ts

```diff
@@ -829,7 +829,73 @@ export class SharedArrayClass<T extends SerializableTypeForSharedArray>
 		deadEntry.isDeleted = true;
 	}
 
-	protected applyStashedOp(_content: unknown): void {
-		throw new Error("Not implemented");
+	protected applyStashedOp(content: unknown): void {
+		const op = content as ISharedArrayOperation<T>;
+
+		switch (op.type) {
+			case OperationType.insertEntry: {
+				this.handleInsertOp<SerializableTypeForSharedArray>(
```

Comment:
Overall, it feels not-ideal that there's not a direct mapping of op to normal-ish local method (e.g. like in MapKernel "clear" -> this.clear(), "delete" -> this.delete(), etc.).  Not saying that treating the stashed op as remote won't have the correct ultimate effect necessarily, but it's hard to inspect with confidence and understand the implications.

I don't want to hold this back as a first implementation, but if we find bugs e.g. when adding it to stress/fuzz or more tests, probably should consider refactoring the DDS impl to allow for more-obvious mapping of stashed op to recreation of the local modifications that got it there rather than simulating remote changes.